packages/dd-trace/src/exporters/common/request.js

```diff
@@ -22,13 +23,17 @@ function request (data, options, keepAlive, callback) {
     options.headers = {}
   }
 
+  const isReadable = data instanceof Readable
+
   // The timeout should be kept low to avoid excessive queueing.
   const timeout = options.timeout || 2000
   const isSecure = options.protocol === 'https:'
   const client = isSecure ? https : http
   const dataArray = [].concat(data)
 
-  options.headers['Content-Length'] = byteLength(dataArray)
+  if (!isReadable) {
+    options.headers['Content-Length'] = byteLength(dataArray)
```

Comment:
Any way to know the content length even with a stream? It's unclear if this was added because of an issue in the past or just as best practice.

Reply:
not that I know